tools/jenkins/include/patches/Qt/vfxplatform/qtbase-015-216facc9693e34eb520b42e851ea55b8a3846743.diff

```diff
@@ -107,7 +107,7 @@ index bce0455..2d49442 100644
 +    if (!m_leftButtonRetained)
 +        m_leftButtonRetained = true;
 +
-     // Handle any active poup windows; clicking outisde them should close them
```

Comment:
this will break the patch, since the patch context is changed here. Please do not fix any typo in that  tools/jenkins/include/patches directory

Reply:
sorry, didn't notice it was a `.diff` will remove.